layer/tests/tests_capabilities.cpp

```diff
@@ -33,17 +33,10 @@ static const char* CONFIG_PATH = "bin/Release";
 static const char* CONFIG_PATH = "lib";
 #endif
 
-struct TestInit {
-    TestInit() : instance(VK_NULL_HANDLE), physical_device(VK_NULL_HANDLE) {}
 
-    ~TestInit() {
-        if (this->instance != VK_NULL_HANDLE) {
-            vkDestroyInstance(this->instance, nullptr);
-            this->instance = VK_NULL_HANDLE;
-        }
-    }
-
-    void init() {
+class profiles : public VkTestFramework {
```

Comment:
How is this working? 

The previous design aimed at allocating only once the "test infrastructure" to ensure the excuting time remains small.

Is `profiles` allocated only once for this file?